extension/src/content.js

```
@@ -245,30 +226,253 @@
+      break;
+      
+    case 'amazonMusic':
+      // Amazon Music: Find the best location in the player bar
+      // Strategy 1: Look for the volume control area (right side of player)
+      const volumeControl = document.querySelector('[aria-label*="ボリューム"], [aria-label*="Volume"], [class*="volume"], #volumeSlider');
+      if (volumeControl) {
+        // Insert before volume control
+        const volumeContainer = volumeControl.closest('div');
+        if (volumeContainer) {
+          volumeContainer.insertAdjacentElement('beforebegin', discordButton);
+          console.log('trunecord: Button inserted before Amazon Music volume control');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)